go/vt/mysqlctl/filebackupstorage/file.go

```diff
@@ -49,79 +53,112 @@ func init() {
 	servenv.OnParseFor("vttablet", registerFlags)
 }
 
-// FileBackupHandle implements BackupHandle for local file system.
-type FileBackupHandle struct {
-	fbs      *FileBackupStorage
+// fileBackupHandle implements BackupHandle for local file system.
+type fileBackupHandle struct {
```

Comment:
I like this but will revert it so that I'm being consistent about not breaking any APIs.